spec/policies/course_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe CoursePolicy do
+
+  let(:user) { User.new }
+  let(:course) { create(:course) }
+
+  subject { described_class }
+
+  permissions ".scope" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.